.browserslistrc

```diff
@@ -0,0 +1 @@
+chrome > 0, edge > 0, and_chr > 0, firefox > 0, opera > 0 and last 1 year
```

Comment:
this finds chrome, edge, firefox, opera, and android chrome (for kiwi) versions in the last year

Reply:
can this be only chrome, and_chr and firefox? opera and edge are chromium anyway